src/test/java/com/epam/ld/module2/testing/MailServerTest.java

```diff
@@ -1,4 +1,33 @@
 package com.epam.ld.module2.testing;
 
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
 public class MailServerTest {
+
+    @Test
+    public void checkSuccessfulSendMessage() {

```

Comment:
Tests names should be a bit more specific, like `send_shouldSendSuccessfully_whenPassedValidAddressAndTemplate`